common/src/geometry/CMakeLists.txt

```diff
@@ -0,0 +1 @@
+add_subdirectory(creator EXCLUDE_FROM_ALL)
```

Comment:
why a lib for geometry ?

Reply:
it uses NSC to precompile to SPIRV embed resources so cannot be header, also it shares utils between 2 examples now hence library (optional and not linked to global common api interface target, if an example want to use it it must link it)